### PR TITLE
Clarify container deletion, fix #172

### LINKED ISSUE #172: Note that only empty containers can be deleted
I added that note about this [in the WAC spec](https://github.com/solid/web-access-control-spec/pull/48/files#diff-04c6e90faac2675aa89e2176d2eec7d8R465) but it actually belongs here in the [api-rest](https://github.com/solid/solid-spec/blob/master/api-rest.md) document.

The LDP spec seems to leave this out of scope.

This implies a 'no' to the question of should deleting a container also delete all its members and members of members (descendants).

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

**Comment by michielbdejong** [comment]
@fabiancook can we merge this, making the current spec like `rmdir`, and then you can open an issue for your proposal to make it like `rm -rf`?

**Comment by fabiancook** [comment]
This seems like it defeats the purpose of the discussion we're having on the issue. By leaving this off, there is no spec change for the addition, it's a clarification, where this is a restriction? 

All but one implementation referenced would be not up to spec given this PR

**Comment by michielbdejong** [comment]
OK so then we disagree not only on what the desired situation is but also on what the current situation is. :) Let's discuss that on #172.

File: api-rest.md [hunk 1]
```diff
@@ -235,6 +235,9 @@ RESPONSE:
 HTTP/1.1 201 Created
 ```
 
+### Deleting containers
+A container can only be deleted if it contains neither member resources nor ACL documents.
```
Comment (acoburn) [inline api-rest.md]: I believe you mean "contained" resources here. "Member resources" has a very different meaning in LDP.

**Comment by michielbdejong** [comment]
@RubenVerborgh @acoburn I think this should address both your comments, can you re-review?

**Comment by michielbdejong** [comment]
great, thanks!

**Pull request merged**